### PR TITLE
Added image url field

### PR DESCRIPTION
**Pull request by naderak**

### CODE REVIEW

File: src/main/java/dk/kb/ds/cumulus/export/CumulusExport.java [hunk 1]
```diff
@@ -61,9 +61,10 @@ public static void main(String[] args) throws Exception {
                 String keyword = record.getFieldValueOrNull("Categories");
                 String subject = record.getFieldValueOrNull("Emneord");
                 String license = record.getFieldValueOrNull("Copyright Notice");
+                String image_url = record.getAssetReference("Asset Reference").getPart(0).getDisplayString();
```
Comment (tokee) [inline src/main/java/dk/kb/ds/cumulus/export/CumulusExport.java]: Opgaven er ikke løst. Det er backends opgave at levere en URL til frontend og det sker ikke her. Den relevante del af `Asset Reference` skal kobles sammen med konfigurerbare URL-stumper, så linket kan bruges uden specialkendskab til hvordan billedserveren fungerer.

**Review by tokee — CHANGES_REQUESTED** [review]

File: src/main/java/dk/kb/ds/cumulus/export/CumulusExport.java [hunk 2]
```diff
@@ -74,6 +81,9 @@ public static void main(String[] args) throws Exception {
                         fieldElement.appendChild(document.createTextNode(attributeContent[i]));
                     }
                 }
+                if (counter > 100)
+                    break;
+                counter++;
```
Comment (tokee) [inline src/main/java/dk/kb/ds/cumulus/export/CumulusExport.java]: Could you remove this or make it an option in the settings?

File: src/main/java/dk/kb/ds/cumulus/export/ImageUrl.java [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+import java.net.HttpURLConnection;
+import java.net.URL;
+
+public class ImageUrl {
+
+    public static String makeUrl (String image_url) {
+        String path1[] = image_url.split(".tif");
+        String path2[] = path1[0].split(":/");
+        String path = path2[1].replace("Depot/DAMX/Online_Master_Arkiv", "DAMJP2/online_master_arkiv");
+        image_url = "https://kb-images.kb.dk/" + path + "/full/!345,2555/0/native.jpg";
```
Comment (tokee) [inline src/main/java/dk/kb/ds/cumulus/export/ImageUrl.java]: This should be a setting, but let's keep it for now to move things forward and make a JIRA instead.

**Review by tokee — CHANGES_REQUESTED** [review]
Unit test run, but no `image_url`s are written to the Solr XML output. It seems that none of the generated URLs exists? for example http://kb-images.kb.dk/Depot/DAM/test/Samlingsbilleder/0000/375/526/KE030219/full/!345,2555/0/native.jpg

Addendum: Solved by switching from test to prod Cumulus.

**Review by danege — COMMENTED** [review]
Is it correct that maven-compiler-plugin version should be 3.2 and not 3.8.1?

File: src/main/java/dk/kb/ds/cumulus/export/CumulusExport.java [hunk 4]
```diff
@@ -32,6 +33,12 @@
     public static void main(String[] args) throws Exception {
 
         try (CumulusServer server = new CumulusServer(Configuration.getCumulusConf())) {
+            Properties exportpro = new Properties();
+
+            exportpro.load(CumulusExport.class.getClassLoader().getResourceAsStream("cumulusExport.properties"));
+            boolean limited = Boolean.parseBoolean(exportpro.getProperty("limited"));
+            int counter = Integer.parseInt(exportpro.getProperty("counter"));
+
```
Comment (tokee) [inline src/main/java/dk/kb/ds/cumulus/export/CumulusExport.java:41]: Property loading and general handling should be part of the already existing `Configuration.java`.

**Review by tokee — CHANGES_REQUESTED** [review]
Why introduce a new property file `cumulusExport.properties` with the Java `Properties` key-value format, instead of expanding the existing `ds-cumulus-export.yml`?

File: src/main/java/dk/kb/ds/cumulus/export/CumulusExport.java (hunk 4 above)
Comment (naderak) [reply src/main/java/dk/kb/ds/cumulus/export/CumulusExport.java:41]: I think a property file is easier to work with than a yml file.

**Pull request merged**